packages/frontend/src/components/PinnedItemsPanel/index.tsx

```diff
@@ -10,11 +14,27 @@ import {
 
 interface Props {
     projectUuid: string;
+    organizationUuid: string;
 }
 
-const PinnedItemsPanel: FC<Props> = ({ projectUuid }) => {
+const PinnedItemsPanel: FC<Props> = ({ projectUuid, organizationUuid }) => {
     const { data: dashboards = [] } = useDashboards(projectUuid);
     const { data: savedCharts = [] } = useSavedCharts(projectUuid);
+    const { user } = useApp();
+
+    const headerTitle = user.data?.ability.can(
+        'update',
+        subject('Project', { organizationUuid, projectUuid }),
+    )
+        ? 'Pinned items'
+        : 'Pinned for you';
+
+    const headerIconTooltipContent = user.data?.ability.can(
+        'update',
+        subject('Project', { organizationUuid, projectUuid }),
```

Comment:
can you extract permission check as a variable and then use that variable above for both headerTitle and tooltipContent picking?